css/index.css

```diff
@@ -86,11 +93,37 @@ h1, h2, h3, h4, h5 {
     margin-top: 16px;
 }
 
+@media ( max-width: 800px ) {
+    .nav {
+        display: flex;
+        align-items: center;
+        margin-top: 16px;
+        flex-wrap: wrap-reverse;
+        justify-content: space-between;
+    }
+}
+
 .nav a {
 	text-decoration: none;
 	color: lightgray;
 }
 
+@media ( max-width: 800px ) {
+    .nav a {
+        text-decoration: none;
+        color: lightgray;
+        margin: 18px;
```

Comment:
Try a % based margin here for the margin-left and margin-right values and it may help with the fluidity.